bullyBot.py

```diff
@@ -0,0 +1,47 @@
+#bullyBot.py
+import os
+import discord
+from discord.ext import commands
+from dotenv import load_dotenv
+
+load_dotenv()
+TOKEN = os.getenv('DISCORD_TOKEN')
+GUILD = os.getenv('DISCORD_GUILD')
+
+bot = commands.Bot(command_prefix='!')
+
+@bot.event
+async def on_ready():
+    for guild in bot.guilds:
+        if guild.name == GUILD:
+            break
+
+    print(
+        f'{bot.user.name} is connected to the following guild:\n'
+        f'{guild.name}(id: {guild.id})'
+    )
+
+@bot.command(name='bully', help='Creates a new channel for bullying a new member')
+async def on_message(ctx, victim):
+    
+
+
+    victim=victim.lower().strip()
+    channel_name=f'making-fun-of-{victim}'
+    guild = ctx.guild
+    existing_channel = discord.utils.get(guild.channels, name=channel_name)
+    bullying_category = discord.utils.get(guild.categories, name='bullying')
+
+    if not existing_channel:
```

Comment:
TODO: swap this out for a fuzzy search (low importance). +1

Reply:
Added todo comment in code

Reply:
Usually TODO comments literally have "TODO" in their text.  That way you can do a text search for them to find all of them.  
+1